Updated FAQ with link to Content migration. (#1787)
umbraco/UmbracoDocs: Umbraco8FAQ.md

With the initial launch of Umbraco 8.0 our main focus has been for you to be able to start using Umbraco 8 when creating a new project. Thus, start a great project from scratch using all the new Umbraco 8 features and functionalities.  

> **It is possible to do a manual content migration** of your Umbraco 7 project to Umbraco 8. For you to able to do so you will need to follow our documentation https://our.umbraco.com/Documentation/Getting-Started/Setup/Upgrading/migrating-to-v8

Reviewer: ```suggestion
**It is possible to do a content migration** of your Umbraco 7 project to Umbraco 8. For you to be able to do so, follow our [Content Migration guide](https://our.umbraco.com/Documentation/Getting-Started/Setup/Upgrading/migrating-to-v8)
```
